src/domain/safe/entities/schemas/transaction-type.schema.ts

```diff
@@ -1,4 +1,5 @@
 import { Schema } from 'ajv';
+import { buildPageSchema } from '../../../entities/schemas/page.schema.factory';
 
 export const transactionTypeSchema: Schema = {
```

Comment:
This is out of the context of this PR, but what about renaming `transactionType*` to simply `transaction*`?

This `TransactionTypeValidator` is used only [here](https://github.com/safe-global/safe-client-gateway-nest/blob/efe7832c9bef6446f6e630058aa70f1522c5a6d9/src/domain/safe/safe.repository.ts#L281) to validate `Transaction` items 🤔